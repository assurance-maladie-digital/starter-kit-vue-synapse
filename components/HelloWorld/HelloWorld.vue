<script setup lang="ts">
const msg = 'Hello World!'
</script>

<template>
	<div>
		<h1 class="text-primary">{{ msg }}</h1>
	</div>
</template>
