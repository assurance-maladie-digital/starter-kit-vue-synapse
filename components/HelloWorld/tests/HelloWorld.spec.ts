// @vitest-environment nuxt
import HelloWorld from '../HelloWorld.vue'
import { mountSuspended } from '@nuxt/test-utils/runtime'
import { describe, it, expect } from 'vitest'

describe('HelloWorld', () => {
	it('should render', async () => {
		const wrapper = await mountSuspended(HelloWorld)

		expect(wrapper.html()).toMatchSnapshot()
		expect(wrapper.text()).toContain('Hello World!')
	})
})
