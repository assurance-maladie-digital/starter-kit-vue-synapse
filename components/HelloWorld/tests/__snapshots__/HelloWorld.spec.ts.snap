// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HelloWorld > should render 1`] = `
"<div>
  <h1 class="text-primary">Hello World!</h1>
</div>"
`;
